packages/api/endpoints/granules.js

```diff
@@ -189,6 +192,9 @@ const putGranule = async (req, res) => {
   }
 
   try {
+    if (isNewRecord === true && !apiGranule.published) {
+      apiGranule.published = false;
+    }
```

Comment:
Thinking out loud... we default `published = false` whenever we're creating a granule (with POST or update) via the API endpoints. 

We do NOT default to `published = false` if we're creating a granule via workflow message or if we're migrating using the upsert logic in `upsertGranuleWithExecutionJoinRecord`.

I'm thinking we have it defaulting on the API endpoints but not the other avenues for creating Granules because it's not possible that we'll have an undefined `published`? But we can't guarantee that so I'm curious what you're thinking is here. Not saying anything is wrong, just trying to understand.

Reply:
Ok so you are defaulting `published = false` in the message write logic https://github.com/nasa/cumulus/pull/3112/files#diff-705c5c617b45462fde4dd4c0b8b964f95e91ec857b774841daebe5144d9dd990R873-R876. Cool. 

What about migrations? I think `published` can be null/undefined in Dynamo so do we want to default there too? Do we already and I just don't see it?

Reply:
Discussed in huddle, agreed migration isn't impacted here and other considerations are out scope.